docs/index.Rmd

```diff
@@ -6,95 +6,140 @@ output:
+
+`cloud`: clouds often appear as you would expect: white or wispy. It's totally possible that the *entire* mission-date you have selected is completely cloudy.
+
+<center>![](www/images/cloud.png){width="400"}</center>
+
+Clouds can look green or even black, too:
+
+<center>![](www/images/cloudgb.png){width="400"}</center>
+
+Sometimes the clouds are barely discernible, but the scene looks 'hazy'. In this case, don't label the haze, but do try to avoid it as you label other classes.
+
+<center>![](www/images/haze.png){width="400"}</center>
+
+`shorelineContamination`: usually dark pixels or yellow-brown pixels that overlap with the shoreline. The purpose of this label is for us to be able to add 'uncertainty' to some machine-inferred labels if they are also proximate to pixels labeled as `shorelineContamination`. Generally speaking, label things that might be 'confusing' for a machine to label because the color is similar to colors of a sediment plume. The easiest way to detect this is to turn on the 'Satellite' baselayer in the top right and toggle the 'Layer 1' option under 'Layers':
+
+<center>![](www/images/sat_toggle.png){width="400"}</center>
+
+<center>![](www/images/contamination.png){width="400"}</center>
+
+<center>![](www/images/contamination_under.png){width="400"}</center>
+
+`other`: anything else that is present in the image that might be classified as 'other' or 'unknown'.
 
 ## How to label: Part 1
 
 The purpose of this step is to walk you through how to setup a script in GEE to be able to open an image before you actually label it. You will need to do this process before each new mission-date combination you are classifying.
 
-1. Go to the project [Github page.](https://github.com/rossyndicate/Superior-Plume-Bloom/tree/main/eePlumB/1_user_interface_script) 
+1.  Go to the project [Github page.](https://github.com/rossyndicate/Superior-Plume-Bloom/tree/main/eePlumB/1_user_interface_script)
+
+2.  Copy the appropriate `.js` script.
+
+    -   If this is your first time through the tutorial, you will be working with the validation script: [`eePlumB_validation.js`](https://github.com/rossyndicate/Superior-Plume-Bloom/blob/5e9d0cd1ad435aa3bdb2f0d2b41c026c0f57c8ab/eePlumB/1_user_interface_script/eePlumB_validation). The purpose of this step is to have you go through the full process of classifying a few images for practice and to provide us with data to see how similarly all our volunteers classify a range of conditions. We hope that the examples below will help guide everyone to classify conditions similarly, but we understand that there will be variation as there is a level of subjectivity in this process.
+    -   If you have already completed the tutorial once using the validation script, you have made it to the main classification workflow and will be working with the classification script: [`eePlumB_classification.js`](LINK%20ME)
```

Comment:
Are we at the point where we can link to the appropriate file and replace where I just wrote `LINK ME`??

Reply:
Absolutely we are, I didn't make it past the 'What are we labeling'. :) I'll be working on the remainder today!

Reply:
Updated in recent commits!